src/plays/savings-calculator/components/SavingOptions/index.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from "react";
+import { calculateSavings } from "../../utils/calculateSavings";
+import "./savingOptions.scss";
+
+function SavingOptions(props) {
+  const { currency, setCurrency, setTotal } = props;
+
+  const [startingBalance, setStartingBalance] = useState(1000);
+  const [monthlyContribution, setMonthlyContribution] = useState(100);
+  const [period, setPeriod] = useState(1);
+  const [periodChoice, setPeriodChoice] = useState("Years");
+  const [annualInterest, setAnnualInterest] = useState(8);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const total = calculateSavings({
+      startingBalance,
+      monthlyContribution,
+      period,
+      periodChoice,
+      annualInterest,
+    });
+
+    setTotal(total);
+  };
+
+  return (
+    <div className="savingOptions">
+      <div className="savingOptions__currencies">
+        <div
+          className={`savingOptions__currency ${
+            currency === "INR" ? "savingOptions__currency--selected" : ""
+          }`}
+          onClick={() => setCurrency("INR")}
+          data-testid="currency-inr"
+        >
+          <h4>INR</h4>
+        </div>
+        <div
+          className={`savingOptions__currency ${
+            currency === "USD" ? "savingOptions__currency--selected" : ""
+          }`}
+          onClick={() => setCurrency("USD")}
+          data-testid="currency-usd"
+        >
+          <h4>USD</h4>
+        </div>
+      </div>
```

Comment:
@kd100100 , Use the select element with options?

Reply:
Modified the tags and made necessary changes!